src/legacy/legacy-default-xpring-client.ts

```diff
@@ -42,9 +40,18 @@ class LegacyDefaultXpringClient implements XpringClientDecorator {
    */
   public static defaultXpringClientWithEndpoint(
     grpcURL: string,
+    forceHttp = false,
```

Comment:
I'm not sure I love this name. This is basically a toggle to force web compatibility right? I'd name it something like 'forceWebCompatibility'.  Curious if @hbergren  has thoughts here. 

Regardless, please document this parameter.

Reply:
@Stormtv , you're saying we only need this parameter to let us test the `web` protocol buffers in Node?

I wonder if it's worth writing Puppeteer tests for the `web` stuff, and just having Node run Node and web run web. (Or not writing Puppeteer tests, and just writing down that we know we should do that.)

Reply:
@hbergren I think that would be the best approach here, I'll document that and implement browser based testing in a later PR.